docs-md/developer-guide/syntax-reference.md

```diff
@@ -398,14 +398,16 @@ message key by setting the `KEY` property of the `WITH` clause.
 Example:
 
 ```sql
-CREATE TABLE users (registertime BIGINT, gender VARCHAR, regionid VARCHAR, userid VARCHAR)
+CREATE TABLE users (rowkey INT KEY, registertime BIGINT, gender VARCHAR, regionid VARCHAR, userid INT)
```

Comment:
all the examples have added `ROWKEY <TYPE> KEY` - is this now a requirement or is it just illustrative?

Reply:
It's not currently required. If you don't add one KSQL currently defaults to `ROWKEY STRING KEY`.  However, we should encourage people to be explicit, rather than rely on implicit.

I've not documented this yet as its a bit meh.  Ideally, if you don't supply the key column it should mean there is no key column.  We can update the docs when this is the case.